LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
+		<skos:definition>mortgage contract that pays out money to the borrower against a set principal limit that is based on the value of existing equity in the underlying collateral.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>A reverse mortgage and an open end loan both have a credit limit.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>The interest is added to the principal balance.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;RightToBuy">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Right To Buy</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;UniversalLoanIdentifier">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;FinancialInstrumentIdentifier"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-be-le-lei;LegalEntityIdentifier"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;identifies"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;Loan"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>universal loan identifier</rdfs:label>
+		<skos:definition>a unique identifier given to unequivocally identify a specific mortgage loan.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>In the US, the structure of this identifier is defined in the 2015 revision to the Home Mortgage Disclosure Act.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;SecondHomeLoan">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;Mortgage"/>
-		<rdfs:label xml:lang="en">second home loan</rdfs:label>
-		<skos:definition xml:lang="en">A loan which may be used for a vacation home or second home used for personal reasons.</skos:definition>
-	</owl:Class>
+	<owl:ObjectProperty rdf:about="&fibo-loan-typ-mtg;assumes">
+		<rdfs:label>assumes</rdfs:label>
+		<skos:definition>relates a new mortgage loan contract to a previous one made to a different borrower for the same property. The new contract is created via a legally binding process where the new borrower assumes the terms of the previous loan contract</skos:definition>
+	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-loan-typ-mtg;isLienOn">
-		<rdfs:label>is lien on</rdfs:label>
+	<owl:ObjectProperty rdf:about="&fibo-loan-typ-mtg;hasAppraisedValue">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasNotionalAmount"/>
```

Comment:
I don't think it's a notional amount as used elsewhere. For a mortgage the notional amount will be the amount borrowed won't it?

Reply:
That was there - I think MU had it in the original.  I've changed it to hasMonetaryAmount.